CanvasColorSpaceProposal.md

```diff
@@ -171,6 +193,7 @@ interface ImageData {
 * putImageData() performs a color space conversion to the color space of the destination canvas.
 * Data returned by getImageData() or passed to putImageData() are assumed to be in linear color space.
 * If the image data color space is not "srgb" or the storage format is not "uint8", the data passed to putImageData() is assumed to be in linear color space.
+* the `luminanceGain` property is used to map sRGB colors to colors with greater dynamic range, including those in the `rec2100pq` and `rec2100pqn` colorspaces, as specified at (https://www.w3.org/TR/ttml2/#style-attribute-luminanceGain).
```

Comment:
Normalizing sRGB to 1 (initial value of luminanceGain) times 80 = 80 cd/m^2 is spec compliant but suboptimal, and does not reflect current practice. A luminanceGain of 2 would seem more appropriate.